src/Http/Requests/LibraryStoreRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace EscolaLms\HeadlessH5P\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LibraryStoreRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'h5p_file' => 'required||max:100000',
```

Comment:
It's better to use arrays, for now its `||` i don't know if laravel will pass that, cos it looks like ['required', null, 'max:100000']. Its better to add:

```php
'h5p_file' => ['required', 'max:100000'],
```